docs/tools.md

```diff
@@ -0,0 +1,378 @@
+# Containerbase Tools
+
+---
+
+- [Prepare](#prepare)
+  - [prepare-tool](#prepare-tool-command)
+- [Install](#install)
+  - [install-tool](#install-tool-command)
+  - [install-gem](#install-gem-command)
+  - [install-npm](#install-npm-command)
+  - [install-pip](#install-pip-command)
+- [Uninstall](#uninstall)
+  - [uninstall-tool](#uninstall-tool-command)
+  - [uninstall-gem](#uninstall-gem-command)
+  - [uninstall-npm](#uninstall-npm-command)
+  - [uninstall-pip](#uninstall-pip-command)
+
+---
+
+## Prepare
+
+### `containerbase-cli prepare tool` <a name="prepare-tool-command"></a>
+
+#### Alias <a name="Alias-prepare-tool"></a>
+
+`prepare-tool`
+
+#### Description <a name="Description-prepare-tool"></a>
+
+Prepares a tool into the container by creating relevant directories,files and links.
+
+#### Usage <a name="Usage-prepare-tool"></a>
+
+`$ containerbase-cli prepare tool [-d,--dry-run] <tools> ...`
+
+`$ prepare-tool [-d,--dry-run] <tools> ...`
+
+#### Examples <a name="Examples-prepare-tool"></a>
+
+- preparing node
+  ```bash
+  prepare-tool node
+  ```
+- preparing all tools
+  ```bash
+  prepare-tool all
+  ```
+
+<br>
+
+## Install
+
+### `containerbase-cli install tool` <a name="install-tool-command"></a>
+
+#### Alias <a name="Alias-install-tool"></a>
+
+`install-tool`
+
+#### Description <a name="Description-install-tool"></a>
+
+Installs a tool into the container.
+
+Additional child tools supported by `install-tool` includes:
+
+- gem: `bundler`
+- npm: `corepack`, `npm`, `pnpm`
+- pip: `checkov`, `copier`, `hashin`, `pdm`, `pip-tools`, `pipenv`, `poetry`, `uv`
```

Comment:
I think this is too vague.

there is a almost complete list in the custom registry docs.

we should have a central list of official supported tools.

Reply:
removed